drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -103,6 +103,7 @@ struct RenderDataGLES3 {
 	Transform3D inv_cam_transform = Transform3D();
 	Projection cam_projection = Projection();
 	bool cam_orthogonal = false;
+	int32_t camera_visible_layers = 0x00000000;
```

Comment:
```suggestion
	uint32_t camera_visible_layers = 0xFFFFFFFF;
```
Should be a uint32_t and the initialization should match the one in renderer_scene_cull.h